# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mathieu/streamtuner/gcc34.diff
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/local.c.orig	2004-09-12 05:26:32.856351552 +0200
+++ ./src/local.c	2004-09-12 05:27:08.615915272 +0200
@@ -614,6 +614,7 @@
 	case LOCAL_TYPE_OGG:	local_vc_read(stream); 		break;
 #endif
 	default:		/* nop */
+		;
 	}
       
       *streams = g_list_append(*streams, stream);
